packages/calypso-analytics/tsconfig.json

```
@@ -0,0 +1,35 @@
+{
+	"compilerOptions": {
+		"target": "ES5",
+		"lib": [ "es6", "DOM", "DOM.Iterable" ],
```

Regarding https://github.com/Automattic/wp-calypso/pull/39451/files#r381582967 - let's try this:

```suggestion
		"lib": [ "DOM", "ESNEXT" ],
```

[`Object.fromEntries` is available](https://unpkg.com/browse/typescript@3.7.5/lib/lib.es2019.object.d.ts), it's a question of finding the right configuration.